src/modules/dataman/dataman.cpp

```diff
@@ -928,6 +929,13 @@ static int backend_check()
 	return 0;
 }
 
+int dataman_check_is_running(){
+	if (is_running()){
+		return PX4_OK;
+	}
+	return PX4_ERROR;
+}
```

Comment:
Just make this return bool and you can return the `is_running()` output directly. Also the `dm_operations_data.running` should be made an atomic type.